datafusion/physical-plan/src/sorts/sort.rs

```diff
@@ -868,11 +868,12 @@ impl DisplayAs for SortExec {
         match t {
             DisplayFormatType::Default | DisplayFormatType::Verbose => {
                 let expr = PhysicalSortExpr::format_list(&self.expr);
+                let preserve_partioning = self.preserve_partitioning;
```

Comment:
NOTE for reviewers: These changes in `sort.rs` is the only real change for this PR.

Everything else is just update tests either manually or automatically (see PR description for steps I used to update those)